README.md

```diff
@@ -101,7 +101,72 @@ example](examples/pom_file_generation/) for more information.
 
 ## API Reference
 
- You can find the complete API reference at [docs/api.md](docs/api.md).
+You can find the complete API reference at [docs/api.md](docs/api.md).
+
+## Pinning artifacts
+
+`rules_jvm_external` supports pinning artifacts and their SHA-256 checksums into
+a `maven_install.json` file that can be checked into your repository.
+
+By pinning artifact versions, you can get improved artifact resolution and fetch
+speed, since `rules_jvm_external` will use Bazel's download mechanism that
+caches files on their sha256 checksums. It also improves resiliency and
+integrity by tracking the sha256 checksums and original artifact urls in the
+JSON file.
+
+To get started with pinning artifacts, first create an empty
+`maven_install.json` file. Then, specify `maven_install_json` in
+`maven_install`:
+
+```python
+maven_install(
+    # artifacts, repositories, ...
+    maven_install_json = "//:maven_install.json",
+)
+```
+
+By specifying `maven_install_json`, an additional `unpinned_maven` (or
+`unpinned_*`) repo will be created. `unpinned_maven` contains targets fetched by
```

Comment:
It would be better to clarify what does `unpinned_*` mean. Something like `unpinned_<your_maven_install_name>` would help people avoid my mistake. :smile:

Reply:
Reworded this section to make it clearer. PTAL.

Reply:
It looks great, thanks.